mkdocs.yml

```diff
@@ -1,13 +1,43 @@
 site_name: Freqtrade
+nav:
+    - About: index.md
```

Comment:
Can you rename this file to `about.md` - and put a index.md into the docs folder to point to the "real" documentation page?

That way we can redirect people from github to there without them reading through the (probably) not properly formatted docs

Reply:
Didn't find any option for this (I've already tried ...). home page should be index.md for readthedocs :/